qcodes/instrument_drivers/Lakeshore/Model_325.py

```diff
@@ -0,0 +1,251 @@
+            parameter_class=GroupParameter
+        )
+
+        self.add_parameter(
+            'powerup_enable',
+            val_mapping={True: 1, False: 0},
+            parameter_class=GroupParameter
+        )
+
+        self.add_parameter(
+            "output_metric",
+            vals=Enum("current", "power"),
+            val_mapping={
+                "current": "1",
+                "power": "2",
+            },
+            parameter_class=GroupParameter
+        )
+
+        Group(
+            [self.input_channel, self.unit, self.powerup_enable,
+             self.output_metric],
+            set_cmd=f"CSET {self._channel}, {{input_channel}}, {{unit}}, "
+                    f"{{powerup_enable}}, {{output_metric}}",
+            get_cmd=f"CSET? {self._channel}"
+        )
+
+        self.add_parameter(
+            'P',
+            vals=Numbers(0, 1000),
+            get_parser=float,
+            parameter_class=GroupParameter
+        )
+
+        self.add_parameter(
+            'I',
+            vals=Numbers(0, 1000),
+            get_parser=float,
+            parameter_class=GroupParameter
+        )
+
+        self.add_parameter(
+            'D',
+            vals=Numbers(0, 1000),
+            get_parser=float,
+            parameter_class=GroupParameter
+        )
+
+        Group(
+            [self.P, self.I, self.D],
+            set_cmd=f'PID {self._channel}, {{P}}, {{I}}, {{D}}',
+            get_cmd=f'PID? {self._channel}'
+        )
+
+        if self._channel == 0:
+            valid_output_ranges = Enum(0, 1, 2)
+        else:
+            valid_output_ranges = Enum(0, 1)
+
+        self.add_parameter(
+            'output_range',
+            vals=valid_output_ranges,
+            set_cmd=f'RANGE {self._channel}, {{}}',
+            get_cmd=f'RANGE? {self._channel}',
+            val_mapping={
+                "Off": '0',
+                "Low (2.5W)": '1',
+                "High (25W)": '2'
+            }
+        )
+
+        self.add_parameter(
+            'setpoint',
+            vals=Numbers(0, 400),
+            get_parser=float,
+            set_cmd=f'SETP {self._channel}, {{}}',
+            get_cmd=f'SETP? {self._channel}'
+        )
+
+        self.add_parameter(
+            "output",
+            get_cmd="HTR?"
+        )
+
+
+class Model_325(VisaInstrument):
+    """
+    Lakeshore Model 331 Temperature Controller Driver
+    Controlled via sockets
+    """
+
+    def __init__(self, name, address, **kwargs):
+        super().__init__(name, address, terminator="\r\n", **kwargs)
+
+        input_channels = ChannelList(self, "thermometer", Model_325_Channel,
+                                     snapshotable=False)
+
+        for chan_name in ['a', 'b']:
+            channel = Model_325_Channel(self, 'sensor_{}'.format(chan_name),
+                                        chan_name)
+            input_channels.append(channel)
+            self.add_submodule('sensor_{}'.format(chan_name), channel)
```

Comment:
Could `channel.name` be used instead of `'sensor_{}'.format(chan_name)` in order to avoid code duplication?